backend/src/crash/game_handler.py

```diff
@@ -207,6 +204,24 @@ def get_crash_event(self):
     def wait_for_crash(self):
         self.crash_event.wait()
 
+    def update_players_history(self):
+        for ws in self.players:
+            current_player = self.players[ws]
```

Comment:
Better enumerate over the key/vlaue pair instead of breaking up in two statements
```suggestion
    for ws, player in self.players.items()
```

Reply:
Sorry that will break the `current_player` name (renamed to `player`) but I find the term `current` unnecessarily verbose.